.gitignore

```diff
@@ -1,6 +1,7 @@
 /target/
 .direnv
 .envrc
+.idea
```

Comment:
I think this should go into your personal .gitignore, or at least be sent as a separate PR

Reply:
Ack, https://github.com/martinvonz/jj/pull/1598 splits it to another PR.

I can't put it in my personal .gitignore since in some projects I _do_ want to check the folder in - IntelliJ is smart enough to create its own `.idea/.gitignore` to ignore personal settings and only commit "generic" project settings.

Reply:
I think we should have a `jj track` for that (i.e. the opposite of `jj untrack`)

Reply:
Hmm, I thought `untrack` was only used to remove files that are `.gitignore`d from the current commit. How would `track` keep track (😅) of what to "unignore" across changes? 🤔

Reply:
I was thinking that `jj track foo` would add that file or directory even if it was in your `.gitignore`